modules/n1ql_fts/pages/index.adoc

```diff
@@ -0,0 +1,589 @@
+        "score": 0.2577992247497947
+    },
+    {
+        "original_title": "Star Wars: The Force Awakens",
+        "score": 0.24131218853216643
+    },
+    {
+        "original_title": "Rogue One: A Star Wars Story",
+        "score": 0.23058263883123975
+    },
+    {
+        "original_title": "The Star of Bethlehem",
+        "score": 0.2153327229007389
+    },
+    {
+        "original_title": "Star Wars: Episode I - The Phantom Menace",
+        "score": 0.20427070096000927
+    },
+    {
+        "original_title": "Star Wars: Episode III - Revenge of the Sith",
+        "score": 0.18790171387166904
+    },
+    {
+        "original_title": "Star Wars: Episode II - Attack of the Clones",
+        "score": 0.18790171387166904
+    },
+    {
+        "original_title": "Star!",
+        "score": 0.11436285490763314
+    },
+    {
+        "original_title": "The Star",
+        "score": 0.11436285295039623
+    },
+    {
+        "original_title": "War",
+        "score": 0.10125028723815423
+    },
+    {
+        "original_title": "The War",
+        "score": 0.10125028550532937
+    },
+    {
+        "original_title": "The War",
+        "score": 0.10125028550532937
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.0885850854356994
+    },
+    {
+        "original_title": "A Star Is Born",
+        "score": 0.0885850854356994
+    },
+    {
+        "original_title": "A Star for Christmas",
+        "score": 0.0885850854356994
+    }
+    ]
+----
+
+
+*Star Wars* was in first position and now after "improving" our search, it appears just as the 5th result. As expected, adding new fields generated more noise in the score. 
+
+Here is how the overview of *Star Wars* looks like:
+
+[quote,]
+____
+Princess Leia is captured and held hostage by the evil Imperial forces in their effort to take over the galactic Empire. Venturesome Luke Skywalker and dashing captain Han Solo team together with the loveable robot duo R2-D2 and C-3PO to rescue the beautiful princess and restore peace and justice in the Empire
+____
+
+
+And here is the overview of *Robot Chicken: Star Wars*
+
+
+[quote,]
+____
+Fans of Adult Swim's 'Robot Chicken' and the Star Wars movie franchise won't want to miss this collection of 30 sketches. This hilarious compilation features an array of skits -- such as 'Darth Vader's Collect Call' and 'Inside the AT-AT' -- as well as the incredible voice talents of Hulk Hogan, Malcolm McDowell, Conan O'Brien and even the original Luke Skywalker himself, Mark Hamill.
+____
+
+In the Robot Chicken case, *Star Wars* appear both in the title and in the description, that is the reason why it is currently more relevant than the original movie. 
+
+
+=== Manipulating Scores with User Defined Functions
+
+So far, if the user searches for *Star War* (with a missing *S*), he would still get fairly relevant results. However, the top 5 ones are not exactly the most popular Star Wars movies ever created.
```

Comment:
Inclusive language. Try `if users search for *Star War* (with a missing *S*), they would still`

Reply:
done